code/ModelAdmin.php

```diff
@@ -183,6 +185,34 @@ public function Link($action = null)
         return parent::Link($action);
     }
 
+    /**
+     * Get the link for a modelTab on this ModelAdmin.
+     * @throws InvalidArgumentException if $modelTab is not managed by this ModelAdmin class.
+     */
+    public function LinkForModelTab(string $modelTab): string
```

Comment:
any specific reason why not
```suggestion
    public function getLinkForModelTab(string $modelTab): string
```
?

Reply:
I was just keeping it similar to `Link()` - I'm happy to throw a get in front of it if you'd prefer.

Reply:
Done